### DISCUSSION TITLE
deploy/deploy-windows

### DISCUSSION THREAD

**Discussion by giscus** [MeoProject/lx-music-api-server#49] [General]
# deploy/deploy-windows

部署教程 1.下载并且安装Python3.10-3.11 前往Python官网下拉即可找到Python3.11.9的下载链接 图1 然后安装Python 图2 2.下载并且安装Git 点击我加速下载 图1 一路点击Next和Install即可 3.使用Git克隆仓库 4.使用pip或poetry进行依赖项安装 pip poetry 5.启动服务器 如果...

https://apidoc.zcmonety.xyz/deploy/deploy-windows.html

**Comment by ikun0014** [comment]
Test111

**Comment by lh2513** [comment]
宝塔上怎么部署？在线等
